Django 5.2.15 / 6.0.6 breaks TestCookieStorage.test_reset_cookie

This [change](https://github.com/django/django/commit/c807d9c398022d23cb27518fa6ecaf343efb30cf) in Django 6.0.6 (and potentially 5.2.15) breaks assumptions about cookies in the django-forms test suite:

```pytb
_____________________ TestCookieStorage.test_reset_cookie ______________________

self = <tests.wizard.test_cookiestorage.TestCookieStorage testMethod=test_reset_cookie>

    def test_reset_cookie(self):
        request = get_request()
        storage = self.get_storage()('wizard1', request, None)
    
        storage.data = {'key1': 'value1'}
    
        response = HttpResponse()
        storage.update_response(response)
    
        cookie_signer = signing.get_cookie_signer(storage.prefix)
        signed_cookie_data = cookie_signer.sign(storage.encoder.encode(storage.data))
>       self.assertEqual(response.cookies[storage.prefix].value, signed_cookie_data)
E       AssertionError: '{"key1":"value1"}:1wV5xo:Od0IH3izBmd3H38mYLNDTqPjtfgw7oKYMfH2xI2eI-w' != '{"key1":"value1"}:1wV5xo:ToEoQuq5tZ1FMcTI4CHPJSnGhpkaaJoC5jS3qnKfr78
E       - {"key1":"value1"}:1wV5xo:Od0IH3izBmd3H38mYLNDTqPjtfgw7oKYMfH2xI2eI-w
E       + {"key1":"value1"}:1wV5xo:ToEoQuq5tZ1FMcTI4CHPJSnGhpkaaJoC5jS3qnKfr78

tests/wizard/test_cookiestorage.py:40: AssertionError
```

--- a/tests/wizard/test_cookiestorage.py
+++ b/tests/wizard/test_cookiestorage.py
@@ -35,14 +35,16 @@ class TestCookieStorage(TestStorage, TestCase):
         response = HttpResponse()
         storage.update_response(response)
 
-        cookie_signer = signing.get_cookie_signer(storage.prefix)
-        signed_cookie_data = cookie_signer.sign(storage.encoder.encode(storage.data))
-        self.assertEqual(response.cookies[storage.prefix].value, signed_cookie_data)
+        request.COOKIES[storage.prefix] = response.cookies[storage.prefix].value
+        self.assertEqual(
+            request.get_signed_cookie(storage.prefix),
+            storage.encoder.encode(storage.data),
+        )
 
         storage.init_data()
         storage.update_response(response)
-        unsigned_cookie_data = cookie_signer.unsign(response.cookies[storage.prefix].value)
+        request.COOKIES[storage.prefix] = response.cookies[storage.prefix].value
         self.assertJSONEqual(
-            unsigned_cookie_data,
+            request.get_signed_cookie(storage.prefix),
             {"step_files": {}, "step": None, "extra_data": {}, "step_data": {}}
         )

